WordPress/Classes/Models/Post+CoreDataProperties.swift

```diff
@@ -4,13 +4,16 @@ import CoreData
 extension Post {
 
     @NSManaged var commentCount: NSNumber?
+    @NSManaged var disabledPublicizeConnections: Dictionary<NSNumber, Dictionary<String, String>>?
```

Comment:
Nope it doesn't and it looks better, I appreciate the style suggestions since I'm not so proficient in Swift as I am in Objective-C